cfme/tests/intelligence/reports/test_widgets.py

```diff
@@ -76,7 +76,7 @@ def _finalize():
         with update(dashboard):
             dashboard.widgets = default_widgets
     request.addfinalizer(_finalize)
-    dashboard.reset_widgets()
+    reset_widgets()
```

Comment:
@psav I can propose only some temporary widgetastic view only for this button, which will be replaced during cfme/dashboard.py conversion.